src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -110,23 +125,38 @@ class ActivityFeed extends React.Component<Props, State> {
             currentUser: prevCurrentUser,
             feedItems: prevFeedItems,
         } = prevProps;
-        const { feedItems: currFeedItems, activeFeedEntryId } = this.props;
-        const { isInputOpen: prevIsInputOpen } = prevState;
-        const { isInputOpen: currIsInputOpen } = this.state;
+        const { feedItems: currFeedItems, activeFeedEntryId, onShowReplies = noop } = this.props;
+        const { activeEntry, isInputOpen: prevIsInputOpen } = prevState;
+        const { activeEntry: prevActiveEntry, isInputOpen: currIsInputOpen } = this.state;
 
         const hasLoaded = this.hasLoaded(prevCurrentUser, prevFeedItems);
         const hasMoreItems = prevFeedItems && currFeedItems && prevFeedItems.length < currFeedItems.length;
         const didLoadFeedItems = prevFeedItems === undefined && currFeedItems !== undefined;
         const hasInputOpened = currIsInputOpen !== prevIsInputOpen;
         const hasActiveFeedEntryIdChanged = activeFeedEntryId !== prevActiveFeedEntryId;
+        const hasActiveEntryChanged = activeEntry?.id !== prevActiveEntry?.id;
 
         if ((hasLoaded || hasMoreItems || didLoadFeedItems || hasInputOpened) && activeFeedEntryId === undefined) {
             this.resetFeedScroll();
         }
 
-        if (didLoadFeedItems || hasActiveFeedEntryIdChanged) {
```

Comment:
Do we still need to check `didLoadFeedItems`? If not, won't this fail in most circumstances?